app/kafka/consumer.py

```
@@ -0,0 +1,113 @@
+                },
+                "status": "failed",
+                "payment": {
+                    "status": "failed",
+                    "transaction_id": f"txn_{random.randint(1000,9999)}"
+                },
+                "created_at": time.strftime("%Y-%m-%dT%H:%M:%SZ")
+            }
+            db_mongo.orders.insert_one(order_doc)
+            print(f"Orden {order_created.order_id} registrada como fallida.")
+
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)